moviefind/src/js/components/Movie.js

```diff
@@ -1,28 +1,30 @@
-var React =require('react');
+var React = require('react');
 var AppActions = require('../actions/AppActions');
 var AppStore = require('../stores/AppStore');
 
 var Movie = React.createClass({
 
-	render: function(){
+	render: function () {
 		var link = 'https://www.imdb.com/title/' + this.props.movie.imdbID;
-		return( 
-			<div className ="well">
-				<div className ="row">
-					<div className ="col-md-4">
-					<img className="thumbnail" src = {this.props.movie.Poster}/>
+		return (
+			<div className="well">
+				<div className="row">
+					<div className="col-md-4">
+						<img className="thumbnail" src={this.props.movie.Poster} />
 					</div>
```

Comment:
It seems like there are unused props `movie.name` and `movie.info` being rendered. If these props are not intended for this component, please remove them to avoid confusion.